kubernetes/loculus/values.yaml

```diff
@@ -1,4 +1,5 @@
 environment: server
+noindex: true
```

Comment:
Suggest moving this to the preview server yaml

Reply:
And maybe it should be camel case?

Reply:
And lastly - do we actually want to disable this for the main branch? I was imagining it just for everything else

Reply:
Up to you, I think hiding by default is ok, if people want indexing it's a one line change. But I don't mind it the other way round. I also admit I don't know how to make loculus config specific to the main preview but not on other previews.

What do you mean by preview server yaml? Do we have an explicit config key "preview server" already? Feel free to push your suggestion or make a PR into this PR.

It's camel case, it's a single word 🤓 we can also call it `noindexRobotsHeader` to make it self explanatory.

I don't feel strongly about any of this. Happy for you to make a call if you feel like it.

Reply:
I think that it's fine to disable it for the main branch as well because I don't really see valuable content that should be found by a search engine. The useful content are on loculus.org.

Reply:
Yep that was my thinking - and Google would still get confused by us having very similar content on main.loculus and pathoplexus - that's one reason to disable on main. We can just change that later as well though.

Reply:
Do we have a way to make `main` config different from _default_? The default is now _not_ to add the noindex, so indexing on. We can do this later, just wondering how we'd do it

Reply:
To do this we would probably:
- update the [appset](https://github.com/loculus-project/loculus/blob/main/kubernetes/appset.yaml) to expect a `noIndex` value that it would pass on as a helm value
- update https://github.com/loculus-project/argocd_metadata/blob/main/config.json to pass on that value

We don't need to do that now. IMO it would be nice in the fullness of time, potentially with some more tweaks to make it even more clear that it's a demo instance.